portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -0,0 +1,44 @@
+import Button from '../components/Button'
+import UserIcon from '../components/UserIcon'
+import { doc } from '@firebase/firestore'
+
+const ProjectCard = ({ project }) => {
```

Comment:
Lots of aesthetic work `todo` here. I didn't want to go too crazy before I had some more context.